Kudu.Services.Web/Content/Scripts/ProcessExplorer.js

```diff
@@ -920,26 +948,36 @@ function handleProfilingEvents(e, processId) {
         return;
     }
 
+    var iisProfiling = false;
```

Comment:
As we discussed before, we need to show a message to the user when they check the iis profiling checkbox. The message should have 'ok' and 'cancel' buttons and proceed if 'ok' is clicked.
The message should specify what IIS profiling does, and the fact that 'This is a relatively expensive option' and the file needs to be opened in 'PerfView' with a link to where to download it from. Should also most likely specift the 3 minute timeout or whatever timeout we pick.